plugins/gpio/README.md

```diff
@@ -0,0 +1,22 @@
+# GPIO
+
+## Introduction
+
+This plugin sets GPIO outputs either high or low before and/or after an
+update has been deployed.
+
+## Quirk Use
+
+This plugin uses the following plugin-specific quirks:
+
+### GpioPrepare
+
+The GPIO bits to set during the update prepare phase, e.g. `5:high,6:low`.
```

Comment:
There will need to be a lot of caution here about when quirking GPIOs since they are very unlikely to be the same across manufacturers or in many cases even models.  I hope there this plugin ends up not being used very frequently, it's a lot better for manufacturers to set up GPIOs in the correct direction using ACPI tables to start with.

Reply:
Yes, my use case is quirking the MTD device for a specific board. It'll use the HwIDS for sure.